src/fuselage/CCPACSFuselage.h

```diff
@@ -197,12 +210,25 @@ class CCPACSFuselage : public generated::CPACSFuselage, public CTiglRelativelyPo
     /**
      * Create a new section, a new element and connect the element to the "startElement".
      * The new element is placed "Before" the start element.
-     * If there is already a element before the start element, we split the existing segment and insert the new element
-     * between the the two elements.
+     * If there is already an element before the start element, an eta has to be prvidied. This funciton will throw an error without an eta. 
+     * This can only happen when called directly.
+     *
+     * @param startElementUID
+     * @param sectionName
+     */
+    TIGL_EXPORT void CreateNewConnectedElementBefore(std::string startElementUID, std::string sectionName);
+
+    /**
+     * Create a new section, a new element and connect the element to the "startElement".
+     * The new element is placed "Before" the start element.
+     * If there is already an element before the start element, we split the existing segment and insert the new element
+     * between the two elements.
      *
      * @param startElementUID
+     * @param sectionName
+     * @param eta
      */
-    TIGL_EXPORT void CreateNewConnectedElementBefore(std::string startElementUID);
+    TIGL_EXPORT void CreateNewConnectedElementBefore(std::string startElementUID, std::string sectionName, double eta);
```

Comment:
For consistency: I believe for most functions, eta comes before the sectionName argument, right?